src/main/java/Domain/Discounts/ConditionalDiscount.java

```diff
@@ -56,4 +57,10 @@ protected void applyDiscountLogic(ShoppingBasket basket) throws StockMarketExcep
             _discount.applyDiscount(basket);
     }
 
+    @Override
+    public BasicDiscountDto getDto() {
```

Comment:
should we do that later?

Reply:
i don't think we will get to it.
Its not mandatory for miluim.